### ISSUE TITLE
Add name to shields readme

### ISSUE THREAD

**Issue by clementlgl** [catie-aq/cookiecutter_zephyr#26]
I think we should add the shield name on the Readme (for the Shield template). Ex:
```
Shield name: `zest_adapte_click`
```
Cf. [Zest_Adapter_Click Readme](https://github.com/catie-aq/zephyr_zest-adapter-click/blob/35e370b58ee8deb483f507003bad77d69cd0d40d/README.md?plain=1#L10)

**Comment by clementlgl** [comment]
@catie-aq/zephyr I need your opinion !
Wouldn't it be better to include the shield comment in a note instead of just write it?

Shield name: `zest_adapte_click`

or

> [!NOTE]
> Shield name: `zest_adapte_click`

**Comment by bperraudin** [comment]
Yes, it would be great to include the shield name on every shield repository, you're right!

**Comment by Sokario** [comment]
Will be processed in Jira: [RD-236](https://catie-aq.atlassian.net/browse/RD-236)
